ci-operator/step-registry/openshift/e2e/openstack/upi/openshift-e2e-openstack-upi-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: openshift-e2e-openstack-upi
+  steps:
+    pre:
+    - chain: upi-openstack-pre
+    test:
+     # - ref: openshift-e2e-test
+    post:
+    - chain: upi-openstack-post
+    env:
+      OPENSTACK_EXTERNAL_NETWORK: "public"
+      OS_CLOUD: "openstack"
+      OPENSTACK_COMPUTE_FLAVOR: "ci.m1.xlarge"
```

Comment:
Please remove cloud-specific configuration from the workflow; it will be selected automatically based on the cluster_profile you set in `config`

Reply:
yeap. will do

Reply:
done